### ISSUE TITLE
Is it possible to get the which cardFace is active?

### ISSUE THREAD

**Issue by nzsambo** [JungHsuan/rn-gesture-flip-card#24]
Thanks for this library!

How can I retrieve which card face is showing so that I can default to the 'front' on focus?

Thanks!

**Comment by JungHsuan** [comment]
It's not possible for now, but I can add this functionality.

**Comment by anniewey** [comment]
@JungHsuan any chance to release this functionality?

Meanwhile I'm creating my own patch as below, but not sure if i update it correctly. I'm passing the index 1 or 0 based on the faces to `onFlipEnd` callback.

```
diff --git a/node_modules/react-native-gesture-flip-card/gestureFlipView.js b/node_modules/react-native-gesture-flip-card/gestureFlipView.js
index fb80bb2..03391d4 100644
--- a/node_modules/react-native-gesture-flip-card/gestureFlipView.js
+++ b/node_modules/react-native-gesture-flip-card/gestureFlipView.js
@@ -137,7 +137,7 @@ const GestureFlipView = React.forwardRef((props, ref) => {
     }).start(() => {
       isAnimating.current = false;
       flippedValue.current = null;
-      props.onFlipEnd && props.onFlipEnd();
+      props.onFlipEnd && props.onFlipEnd(cardFaceRef.current? 1 : 0);
     });
   }
 
@@ -166,7 +166,7 @@ const GestureFlipView = React.forwardRef((props, ref) => {
     }).start(() => {
       isAnimating.current = false;
       flippedValue.current = null;
-      props.onFlipEnd && props.onFlipEnd();
+      props.onFlipEnd && props.onFlipEnd(cardFaceRef.current? 1 : 0);
     });
   }
```

**Comment by JungHsuan** [comment]
> @JungHsuan any chance to release this functionality?
> 
> Meanwhile I'm creating my own patch as below, but not sure if i update it correctly. I'm passing the index 1 or 0 based on the faces to `onFlipEnd` callback.
> 
> ```
> diff --git a/node_modules/react-native-gesture-flip-card/gestureFlipView.js b/node_modules/react-native-gesture-flip-card/gestureFlipView.js
> index fb80bb2..03391d4 100644
> --- a/node_modules/react-native-gesture-flip-card/gestureFlipView.js
> +++ b/node_modules/react-native-gesture-flip-card/gestureFlipView.js
> @@ -137,7 +137,7 @@ const GestureFlipView = React.forwardRef((props, ref) => {
>      }).start(() => {
>        isAnimating.current = false;
>        flippedValue.current = null;
> -      props.onFlipEnd && props.onFlipEnd();
> +      props.onFlipEnd && props.onFlipEnd(cardFaceRef.current? 1 : 0);
>      });
>    }
>  
> @@ -166,7 +166,7 @@ const GestureFlipView = React.forwardRef((props, ref) => {
>      }).start(() => {
>        isAnimating.current = false;
>        flippedValue.current = null;
> -      props.onFlipEnd && props.onFlipEnd();
> +      props.onFlipEnd && props.onFlipEnd(cardFaceRef.current? 1 : 0);
>      });
>    }
> ```

Yes, I think so. Sorry for keeping you waiting, I will work on it soon!

**Comment by JungHsuan** [comment]
@nzsambo @anniewey The version `1.1.0` is published! Please see the README for more information!
